gosub-bindings/src/gosub_api.c

```diff
@@ -0,0 +1,55 @@
+#include "gosub_api.h"
+
+struct render_tree_t *render_tree_init(const char *html) {
+  struct render_tree_t *render_tree = malloc(sizeof(*render_tree));
```

Comment:
Has this a specific usage, or why are you using malloc and store it on the heap? From my C knowledge, you can initialize the struct like `struct render_tree_t *render_tree`.

Reply:
It doesn’t have to be on the heap but it simplifies the initialization a bit. Otherwise the caller will have to create a render tree struct on the stack and pass it to the init function by address. I don’t really have a strong opinion as to whether or not it should be on the stack or heap since it’s pretty small holding only a few pointers

Reply:
I'd say, when it can be on the stack, it should be on the stack

Reply:
That’s fair enough, I can change it